principal-prop-on-premise/README.md

```diff
@@ -0,0 +1,152 @@
+# Configuring Principal Propagation to an On-Premise System
+
+This sample provides details on how to configure principal propagation when you extend an on-premise system using SAP BTP, Kyma runtime.
+
+## Context
+
+The principal propagation relies on the exchange of the JWT token received in Kyma. The exchanged token is then forwarded to the SAP Cloud Connector and is used to identify the logged-in user.
+
+You can use either XSUAA or SAP Identity Authentication Service (IAS) as an external identity provider with user federation.
+
+The following diagram presents a flow in which a logged-in user fetches their sales data from the on-premise system. Using a standalone SAP application router, an API `sap/com/onprem/mysales` is hosted in the Kyma runtime. This API calls the on-premise system using the Connectivity Proxy module.
+
+![flow](assets/pp-on-prem.drawio.svg)
+
+## Prerequisites
+
+* [SAP BTP, Kyma runtime instance](../prerequisites/README.md#kyma)
+* [Kubernetes tooling](../prerequisites/README.md#kubernetes)
+* [Cloud Connector on your laptop or test system](../prerequisites/README.md#sap-cloud-connector)
+* [Node.js](https://nodejs.org/en/download/) - the Node.js runtime, including the node package manager NPM. Install the **LTS** version.
+* [OpenSSL](https://www.openssl.org/) or another similar tool to generate certificates
+
+## Procedure
+
+### Configuring the Cloud Connector
+
+> ### Note:
+> This section uses the official SAP BTP Connectivity service documentation on [Configuring Principal Propagation](https://help.sap.com/docs/CP_CONNECTIVITY/cca91383641e40ffbe03bdc78f00f681/c84d4d0b12d34890b334998185f49e88.html).
+
+1. [Configure Trusted Entities in the Cloud Connector](https://help.sap.com/docs/connectivity/sap-btp-connectivity-cf/set-up-trust-for-principal-propagation?version=Cloud#loioa4ee70f0274248f8bbc7594179ef948d__configure_trust).
```

Comment:
This document mentions only Neo and Cloud Foundry. It could be misleading for the users. Can we explain that somehow?